modules/auxiliary/server/capture/smb.rb

```diff
@@ -17,9 +17,10 @@ def initialize
       'Description' => %q{
         This module provides a SMB service that can be used to capture the challenge-response
         password NTLMv1 & NTLMv2 hashes used with SMB1, SMB2, or SMB3 client systems.
-        Responses sent by this service have by default a random 8 byte challenge string
-        of format `\x11\x22\x33\x44\x55\x66\x77\x88`, allowing for easy cracking using
-        Cain & Abel (NTLMv1) or John the ripper (with jumbo patch).
+        Responses sent by this service by default use a random 8 byte challenge string.
+        A specific value (such as `1122334455667788`) can be set using the CHALLENGE option,
+        allowing for easy cracking using Cain & Abel (NTLMv1) or John the Ripper
+        (with jumbo patch).
```

Comment:
This isn't actually related to the problem statement in the PR description, but I realized while looking through the module that it's no longer accurate. When we updated the module we switched the default to be random which is more difficult to identify as malicious. These changes reflect the current state of things.